portality/static/js/autochecks.js

```diff
@@ -60,23 +60,66 @@ doaj.autocheckers.KeepersRegistry = class {
     }
 
     draw(autocheck) {
-        let icon = this.ICONS[autocheck.advice];
         let message = this.MESSAGES[autocheck.advice];
 
         let context = JSON.parse(autocheck.context);
         message = message.replace("{service}", context.service);
 
-        let style = this.STYLE[autocheck.advice];
+        const frag = drawIconMessage(
+            autocheck, message,
+            this.ICONS[autocheck.advice],
+            this.STYLE[autocheck.advice]);
 
-        let frag = `<div><span class="icon-container icon-container--${autocheck.advice} icon-container--${style}""><span data-feather="${icon}" aria-hidden="true"></span></span>
-                    ${message} (<a href="${autocheck.reference_url}" target="_blank">see record</a>)</div>`;
         return frag;
     }
 }
 
+doaj.autocheckers.NoNoneValue = class {
+    MESSAGES = {
+        "none_value_found": "This field should not contain the value 'None'.",
+    }
+
+    ICONS = {
+        "none_value_found": "x-circle",
+    }
+
+    STYLE = {
+        "none_value_found": "error",
+    }
+
+    draw(autocheck) {
+        let message = this.MESSAGES[autocheck.advice];
+
+        const frag = drawIconMessage(
+            autocheck, message,
+            this.ICONS[autocheck.advice],
+            this.STYLE[autocheck.advice]);
+
+        return frag;
+    }
+}
+
+function drawIconMessage(autocheck, message, icon, style) {
```

Comment:
Functions shouldn't be declared in the global namespace, they should be attached to the `doaj` global.  So if you want this here it should be

```javascript
doaj.autocheckers.drawIconMessage = function(autocheck, message, icon, style) { ... }
```